components/mpas-ocean/src/Registry.xml

```diff
@@ -1133,8 +1133,8 @@
 					possible_values=".true. or .false."
 		/>
 		<nml_option name="config_thickness_flux_type" type="character" default_value="centered"
-					description="If 'upwind', use upwind to evaluate the edge-value for layerThickness, i.e., layerThickEdgeFlux.  The standard MPAS-O approach is 'centered'."
-					possible_values="'upwind', 'centered'"
+					description="If 'upwind', use upwind to evaluate the edge-value for layerThickness, i.e., layerThickEdgeFlux.  The standard MPAS-O approach is 'centered'. For 'constant', uses constant thickness in time from restingThickness, for linear test problems. Note that these two flags are set together for linearized test cases: config_thickness_flux_type = 'constant' linearizes the thickness equation, and config_disable_vel_hadv = .true. linearizes the momentum equation if there is no assumed mean background velocity."
+					possible_values="'upwind', 'centered', 'constant'"
```

Comment:
Does it make sense to move `config_thickness_flux_type` out of the `wetting_drying` section?

Reply:
@sbrus89 Thanks for pointing this out. Yes, I was thinking about doing this with a PR I haven't opened yet to use `config_thickness_flux_type = 'upwind'` with split-explicit. I suppose we could do that in this PR if @mark-petersen is up for it.

Reply:
@mark-petersen I agree!

Reply:
I agree as well.